### PR TITLE
Update The Guardian's ruleset to v3

### LINKED ISSUE #30: Update Guardian address to v3
The Guardian is going to update their landing page with a v3 address on Thursday. At that time, we'll want to update their onion name to the new v3 address: `xp44cagis447k3lpb4wwhcqukix6cgqokbuys24vmxmbzmaq2gjvc2yd.onion` (already up and running and available via their public metadata endpoint).

### PR DESCRIPTION
**Pull request by emkll**
### Status

Ready for review

Closes #30 

Note that merging this before the directory (securedrop.org/directory) is updated and includes the v3 url will break the `generate-and-sign` script (it will replace the rule with the v2 onion address). The rules provided here were generated manually due to the absence of the v3 entry in the directory.

### Review Checklist

- [ ] Changes (or absence o to `onboarded.txt` are accurate 
- [ ] The file `default.rulesets.TIMESTAMP.gz` has been updated, extracting that file and inspecting the contents of the JSON file produces the expected rules
- [ ] The ruleset has been verified by modifying the HTTPS Everywhere configuration in a Tor Browser instance pointing to `Path Prefix`: `https://raw.githubusercontent.com/freedomofpress/securedrop-https-everywhere-ruleset/30-guardian-v3`
- [ ] `index.html` has been updated using `./update_index.sh`

### CODE REVIEW

File: rulesets/guardian-securedrop-ruleset.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <ruleset name="The Guardian">
 	<target host="theguardian.securedrop.tor.onion" />
 	<rule from="^http[s]?://theguardian.securedrop.tor.onion"
-    to="http://33y6fjyhs3phzfjj.onion" />
+    to="http://xp44cagis447k3lpb4wwhcqukix6cgqokbuys24vmxmbzmaq2gjvc2yd.onion/" />
```
Comment (conorsch) [inline rulesets/guardian-securedrop-ruleset.xml:4]: @emkll mentioned out of band that the trailing slash here is incorrect, we'll need to regenerate (and re-sign).

**Comment by conorsch** [comment]
I've appended a commit here, and so would appreciate another full review, including functional testing, from a team member. Note that since we're updating, rather than adding, a rule, the reviewer must confirm that the v3 URL is used in the redirect. Try opening the dev console (ctrl+shift+v in Tor Browser) and using the Network pane to observe the onion host that's being accessed. On my end, I'm still seeing v2, which may be because the prod ruleset is still active is loaded first.

**Comment by eloquence** [comment]
I replaced the official securedrop.org ruleset update channel URL with the GitHub one, and after a restart, it correctly picked up the new rule:

Screenshot from 2021-02-04 10-42-00

**Comment by conorsch** [comment]
I'm seeing the same: removing the prod ruleset channel causes the test-only channel to win. LGTM!

**Review by eloquence — APPROVED** [review]
Changes look good and work in local testing. Tor Browser implicitly verifies the signature, so I've not performed local verification here, but it would be good to add this to the docs / CI in future.

**Pull request merged**